setup.py

```diff
@@ -120,8 +120,8 @@
         "web3[tester]==6.0.0b7",
         # ** Dependencies maintained by ApeWorX **
         "eip712>=0.2.0,<0.3.0",
-        "ethpm-types>=0.3.16,<0.4",
-        "evm-trace>=0.1.0a16",
+        "ethpm-types>=0.4.0,<0.5",
+        "evm-trace>=0.1.0a17",
```

Comment:
- [ ] Requires merging https://github.com/ApeWorX/evm-trace/pull/35
- [ ] and releasing `evm-trace 0.1.0a17`